<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Captura de partículas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Agregar</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Partícula</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>origen en y: </string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QSpinBox" name="velocidad_spinBox_2">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="10" column="1">
            <widget class="QSpinBox" name="blue_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="8" column="1">
            <widget class="QSpinBox" name="red_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="ox_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="10" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>blue: </string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QSpinBox" name="oy_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="12" column="0" colspan="2">
            <widget class="QPushButton" name="agregar_inicio_pushButton">
             <property name="text">
              <string>Agregar al inicio</string>
             </property>
            </widget>
           </item>
           <item row="9" column="1">
            <widget class="QSpinBox" name="green_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSpinBox" name="id_spinBox">
             <property name="maximum">
              <number>1000000</number>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="dx_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="9" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>green: </string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>destino x:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>origen en x: </string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>destino y:</string>
             </property>
            </widget>
           </item>
           <item row="11" column="0" colspan="2">
            <widget class="QPushButton" name="agregar_final_pushButton">
             <property name="text">
              <string>Agregar al final</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QSpinBox" name="dy_spinBox">
             <property name="maximum">
              <number>500</number>
             </property>
            </widget>
           </item>
           <item row="7" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>color (rgb)</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>velocidad: </string>
             </property>
            </widget>
           </item>
           <item row="8" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>red: </string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>id: </string>
             </property>
            </widget>
           </item>
           <item row="13" column="0" colspan="2">
            <widget class="QPushButton" name="mostrar_pushButton">
             <property name="text">
              <string>Mostrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPlainTextEdit" name="salida"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tabla</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="0">
         <widget class="QLineEdit" name="buscar_lineEdit">
          <property name="placeholderText">
           <string>Nombre de la partícula</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QTableWidget" name="tabla"/>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="buscar_pushButton">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="mostrar_tabla_pushButton">
          <property name="text">
           <string>Mostrar</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="Orden_ID_pushButton">
          <property name="text">
           <string>Ordenar por Id</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Orden_Distancia_pushButton">
          <property name="text">
           <string>Ordenar por distancia</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Orden_Velocidad_pushButton">
          <property name="text">
           <string>Ordenar por velocidad</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Impresión</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView"/>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="pushButton_Dibujar">
          <property name="text">
           <string>Dibujar</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="pushButton_Limpiar">
          <property name="text">
           <string>Limpiar</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Búsqueda</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Búsquedas.</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0" colspan="3">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Ingresa el origen a partir del cual se hará la búsqueda: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Origen en X: </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="spinBox_B_OrigenX">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="pushButton_Mostrar_Busqueda_Profundidad">
             <property name="text">
              <string>Mostrar búsqueda en profundidad.</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Origen en Y:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="4">
            <widget class="QSpinBox" name="spinBox_2_B_OrigenY">
             <property name="maximum">
              <number>10000</number>
             </property>
            </widget>
           </item>
           <item row="1" column="5">
            <widget class="QPushButton" name="pushButton_Mostrar_Busqueda_Amplitud">
             <property name="text">
              <string>Mostrar búsqueda en amplitud.</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Impresión de la búsqueda en amplitud.</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="salida_B_Amplitud"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Impresión de la búsqueda en profundidad.</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="0">
            <widget class="QPlainTextEdit" name="salida_B_Profundidad"/>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="actionAbrir"/>
    <addaction name="actionGuardar"/>
   </widget>
   <addaction name="menuArchivo"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
